src/ontology/cl-edit.owl

```diff
@@ -25945,12 +25946,14 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011679"^^xsd:string) obo:
 AnnotationAssertion(rdfs:label obo:CL_1001561 "vomeronasal sensory neuron"^^xsd:string)
 EquivalentClasses(obo:CL_1001561 ObjectIntersectionOf(obo:CL_0000101 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002255) ObjectSomeValuesFrom(obo:RO_0013007 obo:UBERON_0003367)))
 
-# Class: obo:CL_1001566 (bronchioalveolar stem cells)
+# Class: obo:CL_1001566 (obsolete bronchioalveolar stem cells)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011691"^^xsd:string) obo:IAO_0000115 obo:CL_1001566 "A respiratory stem cell found at the junction of the terminal (conductive) bronchiole and the respiratory bronchiole, which gives rise to alveolar cell types and club cells in response to lung injury."^^xsd:string)
-AnnotationAssertion(rdfs:label obo:CL_1001566 "bronchioalveolar stem cells"^^xsd:string)
-SubClassOf(obo:CL_1001566 obo:CL_0000034)
-SubClassOf(obo:CL_1001566 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000115))
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011691"^^xsd:string) obo:IAO_0000115 obo:CL_1001566 "OBSOLETE A respiratory stem cell found at the junction of the terminal (conductive) bronchiole and the respiratory bronchiole, which gives rise to alveolar cell types and club cells in response to lung injury."^^xsd:string)
```

Comment:
```suggestion
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MP:0011691"^^xsd:string) obo:IAO_0000115 obo:CL_1001566 "OBSOLETE. A respiratory stem cell found at the junction of the terminal (conductive) bronchiole and the respiratory bronchiole, which gives rise to alveolar cell types and club cells in response to lung injury."^^xsd:string)
```
This is super minor - I think we want to add a period after OBSOLETE in the definition. I updated this in the OBO Academy documentation:
https://oboacademy.github.io/obook/howto/obsolete-term/#obsoletion-process-manual

Reply:
it won't let me commit this suggestion, will it let you do it @shawntanzk?

Reply:
hmmm strange cause I used the wizard, oh well, either way I've committed the suggestion :) thanks!

Reply:
Oh interesting, does the wizard add the OBSOLETE to the def? (I never use the wizard, I always use Edit -> Make entity obsolete

Reply:
> guess not too, no worries, ill fix it outside

too bad! The inline commit function is so cool but it doesn't always work and I don't understand why.